spec/models/tslr_claim_spec.rb

```diff
@@ -415,4 +415,13 @@
       end
     end
   end
+
+  describe "submitted" do
+    let!(:submitted_claims) { create_list(:tslr_claim, 5, :eligible_and_submittable, submitted_at: DateTime.now) }
```

Comment:
This can use the `:submitted` trait instead, can't it?

Reply:
Ah, right you are